master/usage/policy-example.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Realistic Policy Demo
+---
+
+Three-tier application: frontend, backend, database
```

Comment:
In general I think we should be moving our demos away from simple 3-tier and towards micro-service based applications.